src/uniprotkb/adapters/uniProtkbConverter.ts

```diff
@@ -56,6 +57,9 @@ export const getEntryTypeFromString = (entryTypeString?: string) => {
   if (entryTypeString.match(/Swiss-Prot|reviewed|^sp\|$|^sp$/gi)) {
     return EntryType.REVIEWED;
   }
+  if (entryTypeString.match(/ORCID$/gi)) {
```

Comment:
Not sure how I feel about this, this doesn't feel like it should be within the uniProtKB converter. And the EntryTypeIcon is starting to be a bit more than really an "Entry" type icon

Reply:
Initially thought it wasn't a good idea and then thought actually it could be that in the future we have "COMMUNITY_ANNOTATION" entries alongside SwissProt, TrEMBL etc..
However I think what is misleading is `EntryType.ORCID` so happy to change it to EntryType.COMMUNITY_ANNOTATION

Reply:
I guess it can be better, yes. Still not a fan (for me an annotation is an annotation, about a sequence, and we might have community annotations on reviewed or unreviewed entries, that's why I'm feeling weird about that being here).

Reply:
I think we can leave it for now - ideally we'd have something like an "icon resolver" and we can pass entry types for different namespaces, etc...